Packs/CofenseTriage/pack_metadata.json

```diff
@@ -10,13 +10,9 @@
     "categories": [
         "Data Enrichment & Threat Intelligence"
     ],
-    "tags": [
-        "Email",
-        "Threat Intelligence"
-    ],
+    "tags": [],
     "useCases": [
         "Phishing",
```

Comment:
Are you sure w should leave phishing here?

Reply:
I think so, it is a very holistic pack.